packages/client/README.md

```diff
@@ -0,0 +1,156 @@
+# nftstorage.link client utilities
+
+Utilities for working with the NFT.Storage IPFS Edge Gateway.
+
+## Install
+
+```console
+npm install nftstorage.link
+```
+
+## Usage
+
+Import the library in your client application like:
+
+```js
+import { getGatewayURL } from 'nftstorage.link'
+```
+
+### `getGatewayURL`
+
+Get a gateway URL, given a CID, CID+path, IPFS path or an IPFS gateway URL. If the status of the nftstorage.link gateway is known to be good (according to the status checker) then return a URL that uses nftstorage.link, otherwise return an URL that uses dweb.link (or the optional passed fallback gateway URL). Status result is cached for **60 seconds** by default.
```

Comment:
Probably worth mentioning that dweb and other fallback gateways are not guaranteed to be functional by this utility?

Reply:
Yes, good idea.